other_collections/impact-observatory-lulc/script.js

```diff
@@ -0,0 +1,54 @@
+//VERSION=3
+
+function evaluatePixel(samples) {
+  let tile_sample = NaN;
+  let tile_dataMask = 0;
+  for (let i = 0; i < samples.length; i++) {
+    if (samples[i].dataMask === 1) {
+      tile_sample = samples[i].lulc;
+      tile_dataMask = 1;
+    }
+  }
+  if (tile_sample === 1) {
```

Comment:
To make it cleaner, instead of these many if statements you could use a `ColorMapVisualizer` class. See this script for example: https://github.com/sentinel-hub/custom-scripts/blob/master/copernicus_services/hrsi-wds-ssc/script.js

Reply:
Ahh yes, will do it in this way.